src/libs/actions/IOU.ts

```diff
@@ -5080,9 +5080,9 @@ function navigateToNextPage(iou: OnyxEntry<OnyxTypes.IOU>, iouType: string, repo
             setMoneyRequestParticipants(participants);
         }
         Navigation.navigate(ROUTES.MONEY_REQUEST_CONFIRMATION.getRoute(iouType, report.reportID));
-        return;
+        // return;
     }
-    Navigation.navigate(ROUTES.MONEY_REQUEST_PARTICIPANTS.getRoute(iouType));
+    // Navigation.navigate(ROUTES.MONEY_REQUEST_PARTICIPANTS.getRoute(iouType));
```

Comment:
Why are we commenting out here? If not required, kindly remove the same.

Reply:
For more information, `navigateToNextPage` is removed yesterday [here](https://github.com/Expensify/App/pull/35833). @Krishna2323 Let's merge main